app/helpers/rubygems_helper.rb

```diff
@@ -54,36 +55,42 @@ def unsubscribe_link(rubygem)
             else
               'display:none'
             end
-    link_to t('.links.unsubscribe'), rubygem_subscription_path(rubygem),
+    link_to t('rubygems.show.links.unsubscribe'),
+      rubygem_subscription_path(rubygem),
```

Comment:
nice.

If this pr is not ready to merge, may make sense to just pull these bug fixes out of here.